Task5-2/webpack.config.js

```diff
@@ -0,0 +1,52 @@
+const HtmlWebPackPlugin = require("html-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const CopyPlugin = require('copy-webpack-plugin');
+
+module.exports = {
+    module:{
+        rules:[
+            {
+                test: /\.html$/,
+                use:[
+                    {
+                        loader: "html-loader",
+                        options: {minimize : true}
+                    }
+                ]
+            },
+            {
+                test: /\.(png|jpg)$/,
+                use:[
+                    'file-loader'
+                ]
+            },
+            {
+                test: /\.scss$/,
+                use: [
+                    "style-loader",
+                    "css-loader",
+                    "sass-loader"
+                ]
+            }
+
+        ]
+    },
+    plugins: [
+        new CopyPlugin([
+            { 
+                from: 'src/images/*', 
+                to: '' ,
```

Comment:
Folder `src` inside `dist` folder looks really weird. Copy images to `images` folder.

Reply:
Done